iOS/BlueComputeApp/.gitignore

```diff
@@ -0,0 +1,65 @@
+# Xcode
```

Comment:
Moved the gitignore to this folder as it was not taking effect at the repo's root directory